distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -300,6 +289,16 @@ type Any
                 True -> handler error_value
                 False -> self
 
+    ## PRIVATE
+
+       Executes the provided handler on a dataflow error, or executes as
+       identity on a non-error value.
+
+       Arguments:
+       - handler: The function to call on this if it is an error value.
+    catch_primitive : (Error -> Any) -> Any
+    catch_primitive self handler = @Builtin_Method "Any.catch_primitive"
+
```

Comment:
Curious - why is this moved?

Reply:
Ok, that sounds like a good reason, I just was wondering if there's anything deeper here :)